tests/main/cgroup-tracking-failure/task.yaml

```diff
@@ -0,0 +1,198 @@
+summary: cases where tracking applications and hooks with cgroups does not work
+details: |
+    Cgroup-based application process tracking is implemented using the systemd
+    DBus API org.freedesktop.systemd1.Manager.StartTransientUnit. While exactly
+    what we need in spirit, in practice there are limitations that have been
+    overcome only starting with systemd version 238, which can forward a
+    request from systemd --user to systemd pid one, to perform some cgroup
+    transitions that the kernel normally does not allow.
+systems:
+    - ubuntu-14.04-*
+    - ubuntu-16.04-*
```

Comment:
Isn't tracking working on 16.04 too actually?

Reply:
It does, this test doesn't show just failure. It shows how things behave.

Reply:
Maybe there should be a comment as well `# tracking works correctly, caveats apply`

Reply:
Done